modular_citadel/code/modules/reagents/chemistry/reagents/fermi_reagents.dm

```diff
@@ -58,15 +54,20 @@
 		return ..()
 	var/hatArmor = 0
 	if(!overdosed)
-		hatArmor = (purity/10)
+		hatArmor = (cached_purity/10)
 	else
-		hatArmor = - (purity/10)
-	if(hatArmor > 90)
+		hatArmor = (cached_purity/10)
+	if(hatArmor > 100)
```

Comment:
100 being the max armor value is a hard no.

Reply:
I'm not sure how this was the case, its back to 90, I might've coped old code by accident. Reverted.